src/sidebar/AccountMenuItem.js

```diff
@@ -9,14 +9,15 @@ export default class AccountMenuItem extends PureComponent {
 		currency: string
 	};
 
+	static contextTypes = {
+        router: () => undefined,
+    };
+
 	switchToAccount = () => {
 		const { token } = this.props;
 		storage.setItem('account', JSON.stringify({ token }));
-		if (window.cordova) {
-			window.location.reload(true);
-		} else {
-			window.location.href = window.BinaryBoot.baseUrl;
-		}
+		this.context.router.push('/');
```

Comment:
this change meant switching accounts will set the app to go the home page, instead of refreshing the page. I think this should be the expected behaviour when switching accounts.

Reply:
@brucebinary if the baseUrl is /beta instead of /, will it set to /beta?

Reply:
@arnabk paths in `this.context.router` happens in the context of the app; so if the app exists in a folder `/beta`, it's root path is still `/`. This is different from using `window.location.href`, which changes the URL.